### DISCUSSION TITLE
Low latency ???!!!

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6464] [General]
I activated the DASH low latency  how can i get the m3u8 for that ???
no idea how this works please explain ( enterprise edition )

**Comment by lastpeony** [comment]
HLS (.m3u8): Apple's HTTP Live Streaming protocol uses .m3u8 files for its manifests.

MPEG-DASH (MPD): MPEG's Dynamic Adaptive Streaming over HTTP uses .mpd files.

Both protocols deliver streaming media over HTTP but differ in manifest file format, codec support, and platform adoption.

So those 2 are different protocols.
